docs/en/trust.rst

```diff
@@ -747,6 +724,198 @@ Each JWT containing a Trust Chain in the JWT headers can be verified over time,
+        Issuer: CN=https://trust-anchor.example.com, O=Example Trust Anchor, C=IT
+        Validity
+            Not Before: Sep 1 00:00:00 2023 GMT
+            Not After : Sep 1 00:00:00 2024 GMT
+        Subject: CN=https://leaf.example.org, O=Leaf, C=IT
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+                    00:af:82:3b:...
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Key Usage: 
+                Digital Signature, Key Encipherment
+            X509v3 Subject Alternative Name: 
+                URI:https://leaf.example.com
+            X509v3 Name Constraints: 
+                Permitted:
+                  URI.1=https://leaf.example.com
+                  DNS.1=leaf.example.com
+            X509v3 CRL Distribution Points: 
+                Full Name:
+                  URI:https://trust-ancor.example.com/crl/leaf.example.org.crl
+
+    Signature Algorithm: sha256WithRSAEncryption
+         5c:4f:3b:...
+
+
+    Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 987654321 (0x3ade68b1)
+    Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN=https://leaf.example.org, O=Leaf, C=IT
+        Validity
+            Not Before: Sep 1 00:00:00 2023 GMT
+            Not After : Sep 1 00:00:00 2024 GMT
+        Subject: CN=https://leaf.example.org, O=Leaf, C=IT
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+                    00:af:82:3b:...
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Key Usage: 
+                Digital Signature, Key Encipherment
+            X509v3 Subject Alternative Name: 
+                URI:https://leaf.example.org
+            X509v3 Name Constraints: 
+                Permitted:
+                  URI.1=https://leaf.example.com
+                  DNS.1=leaf.example.com
+            X509v3 CRL Distribution Points: 
+                Full Name:
+                  URI:https://leaf.example.org/crl/self.crl
+
+    Signature Algorithm: sha256WithRSAEncryption
+         7d:6e:5f:...
+
+
+Federation participants can ensure that their certificates are consistent, enabling interoperability and security across the federation. This approach, enabling X.509 certificate issuance delegation, introduces innovative practices for certificate management using the underlying trust relationships established within the OpenID Federation.
+
+
+X.509 Certificate Revocation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An X.509 Certificate can be revoked by its Issuer.
+When the X.509 Certificate issuer is the Leaf and therefore the X.509 Certificate is about itself, it MUST update its CRL.
```

Comment:
In the case of short-lived certificates such as those issued by the Wallet Provider, it is not necessary to have CRLs. See https://www.ietf.org/archive/id/draft-housley-lamps-norevavail-01.html

Reply:
CRL it should be not mandatory, regarding the possibility to have short lived certificates.
I suggest to have a MUST have CRL for long-lived attestations, such as those with an expiration time superior than 24hours.

Reply:
Done here https://github.com/italia/eid-wallet-it-docs/pull/603/files#diff-b00d0cc50af3f9e721a6b5a70f4939182119581219f886e85833e9a133a51101R922